### DISCUSSION TITLE
switch drivers?

### DISCUSSION THREAD

**Discussion by davidroberthoare** [node-dmx/dmx#128] [Q&A]
Hello - thanks for a great package!

I'm building a simple lighting board app in electron. I can successfully choose either the null, or an enttec-usb-dmx-pro driver at startup.
However, if I try to switch between the drivers while the program is running, I end up with 2 universes...
ex.

```
//at startup
var dmx = new DMX();
var universe = dmx.addUniverse('demo', 'null');

//then later, on a menu option...
dmx = new DMX();
universe = dmx.addUniverse('demo', 'enttec-usb-dmx-pro', found);
```

In the console I can see that the null device is still running and logging buffer output, but the new enttec is also working...

What would be the proper way to stop/remove/destroy the previous universe and start a new one?

**Comment by wiedi** [comment]
You should be able to stop it with `universe.stop()`.
